### ISSUE TITLE
USING() in joins not supported

### ISSUE THREAD

**Issue by BrandiATMuhkuh** [oguimbal/pgsql-ast-parser#30]
It seems like the `USING()` keyword for joins is not supported. At the moment it can be "fixed" by using `"STUD_ASS_PROGRESS"."studentId" = "ACCURACY"."studentId"`

Postgres docs
https://www.postgresql.org/docs/9.2/queries-table-expressions.html

Query:
```
SELECT *
FROM STUD_ASS_PROGRESS
LEFT JOIN ACCURACY
USING("studentId")
```

Error
```
Error: Syntax error at line 40 col 25:
> 
>                            USING
>                            ^
>  Unexpected kw_using token: "using". Instead, I was expecting to see one of the following:
> 
>      - A "dot" token
>      - A "dot" token
>      - A "lparen" token
>      - A "kw_as" token
>      - A "kw_primary" token
>      - A "kw_unique" token
>      - A "word" token
>      - A "kw_on" token
>      - A "kw_left" token
>      - A "kw_right" token
>      - A "kw_full" token
>      - A "kw_inner" token
>      - A "kw_where" token
>      - A "kw_join" token
>      - A "kw_group" token
>      - A "kw_order" token
>      - A "kw_limit" token
>      - A "kw_offset" token
>      - A "kw_fetch" token
>      - A "kw_union" token
>      - A "rparen" token

```

**Comment by oguimbal** [comment]
Hi ! Fixed in `pgsql-ast-parser@7.0.1`. Do you need it implemented in pg-mem ?

**Comment by oguimbal** [comment]
Ops... didnt see that it was a column list, not an expression list... fix published `pgsql-ast-parser@7.0.2`

**Comment by BrandiATMuhkuh** [comment]
wow that was quick! 
No I don't need it in `pg-mem`. But thanks for asking
